events.html

```diff
@@ -228,6 +228,10 @@ <h4 class="modal-title text-center">Use this snippet to send an event to server<
                 }
 
                 data.count = parseInt($("#event-count").val()) || 1;
+                if (data.count < 0) {
+                    $("#event-count").parent().prepend('<div class="alert alert-danger" role="alert">Count for event cannot be negative</div>');
```

Comment:
the scrollToElement (a few lines above) scrolls to the given element. if you remember when user scrolled down to press submit the error was printed at the top of the page where user can not see it. would this do the same?

Reply:
It was not scrolling but it was going up. Thanks for the feedback, scroll obviously looks much cleaner. It's added now